src/objectTranslations/Account-fr.objectTranslation

```diff
@@ -1,52 +1,52 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
-  <fields>
-    <name>Current_Address__c</name>
-    <help>L’adresse actuellement renseignée dans les champs de facturation du compte.</help>
-    <relationshipLabel>Comptes (adresse actuelle)</relationshipLabel>
-    <label>Adresse actuelle</label>
-  </fields>
-  <fields>
-    <name>Primary_Contact__c</name>
-    <relationshipLabel>Compte</relationshipLabel>
-    <label>_SYSTEM: One2OneContact</label>
-  </fields>
-  <layouts>
-    <layout>HEDA Organization Layout</layout>
-    <sections>
-      <label>Champs</label>
-      <section>Fields</section>
-    </sections>
-    <sections>
-      <label>Adresse</label>
-      <section>Address</section>
-    </sections>
-    <sections>
-      <label>Description</label>
-      <section>Description</section>
-    </sections>
-    <sections>
-      <label>Liens personnalisés</label>
-      <section>Custom Links</section>
-    </sections>
-  </layouts>
-  <layouts>
-    <layout>HEDA Household Layout</layout>
-    <sections>
-      <label>Champs</label>
-      <section>Fields</section>
-    </sections>
-    <sections>
-      <label>Adresse</label>
-      <section>Address</section>
-    </sections>
-    <sections>
-      <label>Description</label>
-      <section>Description</section>
-    </sections>
-    <sections>
-      <label>Liens personnalisés</label>
-      <section>Custom Links</section>
-    </sections>
-  </layouts>
+    <fields>
+        <help>L’adresse actuellement renseignée dans les champs de facturation du compte.</help>
```

Comment:
@robertbwatson is this supposed to be a single quote (&pos;) here or is this a different character all together?

Reply:
that comment was supposed to be on the other side but I guess it is in both places anyway :)